### PR TITLE
175-convert-panelrenderers-to-react-components

### LINKED ISSUE #175: Convert `panelRenderers` to React components

### PR DESCRIPTION
**Pull request by jefferymills**

### CODE REVIEW

File: src/actions/index.js [hunk 1]
```diff
@@ -336,9 +336,9 @@ const setCogDatAndState = (iface, cogDatJson, dObjJson, dispatch, hash) => {
 
 const setPanelInfo = (dObjJson, cfg, dispatch) => {
   if (dObjJson.panelInterface.type === 'image') {
-    dispatch(setPanelRenderers(dObjJson.name, (x, width, height) => <img src={x} alt="panel" style={{ width, height }} />));
+    /* dispatch(setPanelRenderers(dObjJson.name, (x, width, height) => <img src={x} alt="panel" style={{ width, height }} />)); */
```
Comment (Incin) [inline src/actions/index.js:339]: did you want this commented out code?

File: src/actions/index.js [hunk 2]
```diff
@@ -347,51 +347,11 @@ const setPanelInfo = (dObjJson, cfg, dispatch) => {
           // style={{ width, height }}
         />
       )),
```
Comment (Incin) [inline src/actions/index.js:349]: same as above

File: src/index.js [hunk 3]
```diff
@@ -227,13 +227,10 @@ window.trelliscopeApp = trelliscopeApp;
...
-trelliscopeApp('80222985', '_test/gapminder_coggroups/config.jsonp', { logger: true });
-// trelliscopeApp('62e90658', '_test/gapminder_bells/config.jsonp', { logger: true });
-// trelliscopeApp('62e90658', '_test/trelliscope-examples2/gapminder_bells/config.jsonp', { logger: true });
-// trelliscopeApp('80222985', '_test/gapminder_coggroups/config.json', { logger: true });
+// trelliscopeApp('80222985', '/config.json', { logger: true, mockData: true });
+// trelliscopeApp('80222985', '_test/gapminder_coggroups/config.jsonp', { logger: true });
 // trelliscopeApp('96c61ca5', '_test/trelliscope-examples2/gapminder_reldisp/config.jsonp', { logger: true });
-// trelliscopeApp('96c61ca5', '/config.json', { logger: true, mockData: true });
-// trelliscopeApp('17a6ca23', '_test/trelliscope-examples2/network_nonraster/config.jsonp', { logger: true });
+trelliscopeApp('17a6ca23', '_test/trelliscope-examples2/network_nonraster/config.jsonp', { logger: true });
```
Comment (Incin) [inline src/index.js]: did you want this commited?

**Review by Incin — APPROVED** [review]

File: src/actions/index.js (hunk 1 above)
Comment (jefferymills) [reply src/actions/index.js:339]: I'm removing it in my current task.

File: src/index.js (hunk 3 above)
Comment (jefferymills) [reply src/index.js]: No

**Pull request merged**